### DISCUSSION TITLE
Recommended!project packaging portability

### DISCUSSION THREAD

**Discussion by 969687200** [Sanster/IOPaint#247] [Ideas]
Engineering package portability, using the command line in the window, refer to the website: https://github.com/xinntao/Real-ESRGAN/blob/master/README.md
This executable file is portable and includes all the binaries and models required. No CUDA or PyTorch environment is  needed.

**Comment by 969687200** [comment]
Refer to this operation mode：https://github.com/1024803482/Inpainting-Demo-Lama

**Comment by Sanster** [comment]
add RealESRGAN and other plugins in 1.0.2:  https://github.com/Sanster/lama-cleaner/releases/tag/1.0.0
